database/migrations.php

```diff
@@ -35,7 +35,14 @@ function migate(mysqli $db)
     CREATE TABLE Items
     (
     id INTEGER AUTO_INCREMENT PRIMARY KEY,
-    username VARCHAR(40) NOT NULL
+    itemName VARCHAR(40) NOT NULL,
+    description VARCHAR(100) NOT NULL,
```

Comment:
We should probably make these longer, as an example "Mario Kart 8: Deluxe Edition" is 28 chars already. We might get away with limiting to 40/50 on the name, but I think description should be at least 255.